queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/aliases.py

```diff
@@ -29,8 +29,9 @@ def update_aliases(business: LegalEntity, aliases) -> Dict:
         ):
             existing_alias.alias = alias["name"].upper()
         else:
-            new_alias = Alias(alias=alias["name"].upper(), type=Alias.AliasType.TRANSLATION.value)
-            business.aliases.append(new_alias)
+            new_alias = Alias(alias=alias["name"].upper(), type=Alias.AliasType.TRANSLATION.value)  # noqa: F841
+            # TODO: Due to model changes, aliases now returns a query object, which cannot be appended to.
+            # business.aliases.append(new_alias)
```

Comment:
Since `business.aliases` is no longer [a relationship](https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/models/business.py#L250), and is now a [computed property that returns a query object](https://github.com/bcgov/lear/blob/feature-legal-name/legal-api/src/legal_api/models/legal_entity.py#L315), we can no longer use `append`.  This is causing the worker unit test (test_incorporation.py) to fail. For the purposes of this ticket we are ignoring alias logic, so I am just going to leave this commented out.